code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -259,3 +259,73 @@
 		E.pixel_y = rand(-6,6)
 		if(animal_count[src.type] < ANIMAL_CHILD_CAP && prob(10))
 			processing_objects.Add(E)
+
+#define BOX_GROWTH_BAR 200
+/mob/living/simple_animal/hostile/retaliate/box
+	name = "box"
+	desc = "A distant descendent of the common domesticated Earth pig, corrupted by generations of splicing and genetic decay."
+	icon_state = "box"
+	icon_living = "box"
+	icon_dead = "box_dead"
+	speak = list("SQUEEEEE!","Oink...","Oink, oink", "Oink, oink, oink", "Oink!", "Oiiink.")
+	emote_hear = list("squeals hauntingly")
+	emote_see = list("roots about","squeals hauntingly")
+	speak_chance = 1
+	turns_per_move = 6
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/box
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	attacktext = "kicks"
+	health = 60
+	melee_damage_lower = 10
+	melee_damage_upper = 12 //Those tusk will maul you!
+	size = SIZE_SMALL
+	min_oxy = 0
+	max_oxy = 1
+	min_n2 = 5
+	max_n2 = 0
+	treadmill_speed = 1.5
+	var/fat = 0
+
+/mob/living/simple_animal/hostile/retaliate/box/proc/updatefat()
+	if(size<SIZE_BIG)
+		size++
+		fat = 0
+
+/mob/living/simple_animal/hostile/retaliate/box/examine(mob/user)
+	..()
+	switch(size)
+		if(SIZE_SMALL)
+			to_chat(user, "<span class='info'>It's a box baby.</span>")
+		if(SIZE_NORMAL)
+			to_chat(user, "<span class='info'>It's a respectable size.</span>")
+		if(SIZE_BIG)
+			to_chat(user, "<span class='info'>It's huge - a prize winning porker!</span>")
+
+/mob/living/simple_animal/hostile/retaliate/box/CanAttack(atom/A)
+	if(ishuman(A))
+		var/mob/living/carbon/human/H = A
+		if(isvox(H)) return 0 //Won't attack Vox
+	else ..()
+
+/mob/living/simple_animal/hostile/retaliate/box/Life()
+	. = ..()
+	if(size<SIZE_BIG) fat += rand(2)
```

Comment:
eh fine, that one's not even a particularly good looking one line if, they're better for returns